### DISCUSSION TITLE
Weakness / improve mode?

### DISCUSSION THREAD

**Discussion by subash774** [monkeytypegame/monkeytype#3724] [Ideas] [answered]
There are a few letters that are usually mistyped (I struggle with v/b t/y p/[), would it be a good idea to count/monitor user mistyped words and repeat them or choose more words with mistyped letters? That way user's typing experience is personalised and they get to improve on their weakness.

**Comment by Miodec** [comment, marked as answer]
You can use the weakspot funbox. A more persisting solution for weak letters and bigrams is planned for the future.
